yw_website/yw_website/apps/website/urls.py

```diff
@@ -28,6 +28,7 @@
          detailed_workflow, name='detailed_workflow'),
     path('run_detail/<int:run_id>/', run_detail, name='run_detail'),
     path('my_workflows/edit/<int:workflow_id>/version/<int:version_id>/', edit_workflow, name='edit_page'),
+    path('my_workflows/<int:workflow_id>', delete, name='delete'),
```

Comment:
django convention is to add a `/` at the end of every route, so lets try to stay consistent with that